### PR TITLE
Dev: add SITL install for Windows with WSL

### PR DESCRIPTION
**Pull request by rmackay9**
This adds a new page with instructions for installing SITL on Windows when using WSL/WSL2.

The [current instructions](https://ardupilot.org/dev/docs/SITL-setup-landingpage.html) are quite misleading to WSL users because it makes no mention of WSL and instead new devs were being misdirected off to the Cygwin setup.

This has been tested locally and looks OK to me.

### CODE REVIEW

**Comment by TunaLobster** [comment]
I agree that the current state of the flow to get to those institutions isn't great. I don't think this does enough to restructure the flow to change the outcome. This whole thing needs a rethink with all of the options (Linux, WSL, VM on Windows, and Cygwin) all thought out with explicit stipulations on what path the user is suggested to take.

The primary issue is that the link to the WSL institutions is hidden in a paragraph at the top of the Cygwin page. The choice between WSL, Cygwin, and VM on Windows should be done and explained before presenting the user with any instructions about what to do.

This might intertwine with the Building the Code pages as well.

The proposed page restructure is this:

Level 1: A simulation landing page included Sim on Hardware and SITL. Also explain that HIL is no longer supported. In the SITL section, include a table of physics simulators that are supported, sections for each host OS with options, explicit suggestions, and niche use cases when other options are valid (i.e. Cygwin).

Level 2: A page for each option (I suppose this is where it would tie in with Building the Code). Direct the user to go to the environment set up page and then return to this page where they are directed how to run SITL, connect a GCS, and talk about any of the other slightly odd things that might come up while they are using the chosen option.

**Comment by Hwurzburg** [comment]
Will review later this week...you don't need Cygwin to create windows sim .exe...any PR creates the artifacts needed for MP sim

File: dev/source/docs/sitl-native-on-windows.rst [hunk 1]
```diff
@@ -4,6 +4,10 @@
 Setting up SITL on Windows
```
Comment (Hwurzburg) [inline dev/source/docs/sitl-native-on-windows.rst:4]:
```suggestion
SITL setup on Windows using Cygwin (not recommended)
```
be sure to fix over/underlines

**Review by Hwurzburg — DISMISSED** [review]
a few tweaks...thanks

File: dev/source/docs/sitl-on-windows-wsl.rst [hunk 2]
```diff
@@ -0,0 +1,122 @@
+.. _sitl-on-windows-wsl:
+
+=========================
+SITL on Windows using WSL
+=========================
+
+Before following these instructions please following :ref:`Setting up the Build Environment using WSL <building-setup-windows>`.  For Windows10 users the sections regarding :ref:`installing XWindows or VcXsrv <building-setup-windows10_XWindows>` are required.
+
+Once thes above has been completed, using SITL under Windows WSL is nearly the same as :ref:`Setting up SITL on Linux <setting-up-sitl-on-linux>`
```
Comment (khancyr) [inline dev/source/docs/sitl-on-windows-wsl.rst:9]: the above or these above maybe ?

File: dev/source/docs/sitl-native-on-windows.rst (hunk 1 above)
Comment (rmackay9) [reply dev/source/docs/sitl-native-on-windows.rst:4]: ok changed title in follow-up commit.

File: dev/source/docs/setting-up-sitl-on-windows.rst [hunk 3]
```diff
@@ -4,6 +4,10 @@
 SITL on Windows in a Virtual Machine - VM (Manual Setup)
```
[the remark this answers is not in the document]
Comment (rmackay9) [reply dev/source/docs/setting-up-sitl-on-windows.rst:4]: ok, changed title.

File: dev/source/docs/sitl-on-windows-wsl.rst [hunk 4]
```diff
@@ -0,0 +1,122 @@
+.. _sitl-on-windows-wsl:
+
+=========================
+SITL on Windows using WSL
+=========================
+
+Before following these instructions please following :ref:`Setting up the Build Environment using WSL <building-setup-windows>`.  For Windows10 users the sections regarding :ref:`installing XWindows or VcXsrv <building-setup-windows10_XWindows>` are required.
```
[the remark this answers is not in the document]
Comment (rmackay9) [reply dev/source/docs/sitl-on-windows-wsl.rst:7]: ok, reworded.

File: dev/source/docs/sitl-on-windows-wsl.rst [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+       :width: 450px
+
+   .. tip::
+
+      It is useful to arrange the windows as shown above, so you can
+      observe the status and send commands at the same time.
+
+#. Send commands to SITL from the command prompt and observe the results on the map.
+
+   -  Change to GUIDED mode, arm the throttle, and then takeoff:
```
[the remark this answers is not in the document]
Comment (rmackay9) [reply dev/source/docs/sitl-on-windows-wsl.rst:49]: ok, fixed.

File: dev/source/docs/sitl-on-windows-wsl.rst [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+Once thes above has been completed, using SITL under Windows WSL is nearly the same as :ref:`Setting up SITL on Linux <setting-up-sitl-on-linux>`
+
+.. image:: ../images/MAVProxy_Map_GuidedCopter.jpg
+   :target: ../_images/MAVProxy_Map_GuidedCopter.jpg
+   :width: 450px
+
+Running SITL
+============
+
+#. Open an Ubuntu terminal or use the :ref:`VSCode <editing-the-code-with-vscode>` Terminal
```
[the remark this answers is not in the document]
Comment (rmackay9) [reply dev/source/docs/sitl-on-windows-wsl.rst:18]: added clarification

File: dev/source/docs/sitl-on-windows-wsl.rst (hunk 2 above)
Comment (rmackay9) [reply dev/source/docs/sitl-on-windows-wsl.rst:9]: fixed, txs

**Comment by rmackay9** [comment]
Thanks very much!

**Pull request merged**